geometry_msgs/PoseStamped pose
float32 v_max
float32 v_min